services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/permissions/HankeKayttajaController.kt

```diff
@@ -378,14 +378,51 @@ Returns the updated hankekayttaja.
     )
     @PreAuthorize("@hankeKayttajaAuthorizer.authorizeHankeTunnus(#hankeTunnus, 'VIEW')")
     fun updateOwnContactInfo(
-        @RequestBody update: ContactUpdate,
+        @RequestBody update: OwnContactUpdate,
         @PathVariable hankeTunnus: String
     ): HankeKayttajaDto {
         return hankeKayttajaService
             .updateOwnContactInfo(hankeTunnus, update, currentUserId())
             .toDto()
     }
 
+    @PutMapping("/hankkeet/{hankeTunnus}/kayttajat/{kayttajaId}")
+    @Operation(
+        summary = "Update the contact information of a user",
+        description =
+            """
+Updates the contact information of the hankekayttaja the user has in the hanke.
```

Comment:
This is not accurate, since this endpoint updates more than contact info and for the hankekayttaja that's specified in the URL.

Reply:
I changed the text